src/components/ArticleInfo.jsx

```diff
@@ -1,27 +1,35 @@
 import { Link } from "react-router-dom"
-import { useEffect, useState} from "react"
+import { useState} from "react"
 import axios from "axios"
 
 export default function ArticleInfo({article}) {
     let [addVote, setAddVote] = useState(0)
+    let {article_id, title, topic, author, created_at, comment_count, votes} = article
 
     function voteUp () {
-        setAddVote(1)
-        axios.patch(`https://nc-news-backend-shahbazrafi.herokuapp.com/api/articles/${article.article_id}`, { inc_votes : 1})
+        setAddVote((x) => x+1)
```

Comment:
Not sure why there's vote functionality in your PR for sort articles?

Reply:
I think it's because you've re-incorporated things from previous tickets into this one. Being in a position where you're going to merge so many branches into main in one go is a recipe for merge-errors.